packages/flowtip-react-dom/src/types.tsx

```diff
@@ -22,25 +13,25 @@ export type Borders = {
 export type ContentProps = {
   style: Style,
   result: Result,
-  children: React.Node,
+  children: React.ReactNode,
 };
 
 export type TailProps = {
   style: Style,
   result: Result,
-  children: React.Node,
+  children: React.ReactNode,
 };
 
 export type RenderProps = {
-  onContentSize(Dimensions): mixed,
-  onTailSize(Dimensions): mixed,
+  onContentSize(Dimensions): any,
+  onTailSize(Dimensions): any,
```

Comment:
Consider using `unknown` instead of the dangerous `any`

https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#new-unknown-top-type

Reply:
👍